docs/component-touchables.md

```diff
@@ -0,0 +1,51 @@
+---
+id: component-touchables
+title: Touchables
+sidebar_label: Touchables
+---
+Basing on [native buttons](component-buttons.md) Gesture Handler exposes reimplementation of touchables, which follows expected behavior of these imported from React Native.
+
+React Native's touchables API could be found here:
+ - [Touchable Highlight](https://facebook.github.io/react-native/docs/touchablehighlight)
+ - [Touchable Opacity](https://facebook.github.io/react-native/docs/touchableopacity)
+ - [Touchable Highlight](https://facebook.github.io/react-native/docs/touchablehighlight)
+ - [Touchable Without Feedback](https://facebook.github.io/react-native/docs/touchablewithoutfeedback)
+ 
+ All important props (but for `pressRetentionOffset`) has been copied and should behave in a similar way to original components. 
+ 
+ The motivation for using RNGH touchables as a replacement for these imported from React Native is to follow native behavior more strictly.
```

Comment:
```suggestion
 The motivation for using RNGH touchables as a replacement for these imported from React Native is to follow built-in native bahavior more closely by utilizing platform native touch system instead of relying on the JS responder system.
```